Library/ViewModels/RootViewModelTests.swift

```diff
@@ -490,28 +490,28 @@ final class RootViewModelTests: TestCase {
     self.vm.inputs.didSelect(index: 0)
 
     self.selectedIndex.assertValues([0, 1, 0], "Selects index immediately.")
-    XCTAssertEqual(["Tab Bar Clicked", "Tab Bar Clicked"], self.dataLakeTrackingClient.events)
+    XCTAssertEqual(["Tab Bar Clicked", "CTA Clicked"], self.dataLakeTrackingClient.events)
     XCTAssertEqual(
-      ["activity", "discovery"],
+      ["activity", nil],
       self.dataLakeTrackingClient.properties(forKey: "context_tab_bar_label")
     )
-    XCTAssertEqual(["Tab Bar Clicked", "Tab Bar Clicked"], self.segmentTrackingClient.events)
+    XCTAssertEqual(["Tab Bar Clicked", "CTA Clicked"], self.segmentTrackingClient.events)
     XCTAssertEqual(
-      ["activity", "discovery"],
+      ["activity", nil],
       self.segmentTrackingClient.properties(forKey: "context_tab_bar_label")
     )
 
     self.vm.inputs.didSelect(index: 10)
 
     self.selectedIndex.assertValues([0, 1, 0, 3], "Selects index immediately.")
-    XCTAssertEqual(["Tab Bar Clicked", "Tab Bar Clicked"], self.dataLakeTrackingClient.events)
+    XCTAssertEqual(["Tab Bar Clicked", "CTA Clicked"], self.dataLakeTrackingClient.events)
     XCTAssertEqual(
-      ["activity", "discovery"],
+      ["activity", nil],
```

Comment:
I guess the reason why it's nil here is because we've removed `discovery` from `context_tab_bar_label`

Reply:
Ah so the real reason is because prior to this PR, we were handling the selection of the `Explore/Discovery` tab in a default case which set the `context_tab_bar_label` value. It is not in the spec for the newly defined event though.